app/v1/__init__.py

```diff
@@ -0,0 +1,102 @@
+
+from flask_api import FlaskAPI
+from flask import request, jsonify, abort
+
+
+# local import configs and routes(route file merged here)
+from instance.config import app_config
+
+
+
+
+
+
+def create_app(config_name):
+    from v1.book import Book
+    app = FlaskAPI(__name__, instance_relative_config=True)
+    app.config.from_object(app_config[config_name])
+    app_config[config_name].init_app(app)
+
+    @app.route('/api/books/', methods=['POST', 'GET'])
+    def booklist():
+        """Adds a book to the library if POST, or retrieves all books if GET is used"""
+
+        if request.method == "POST":
+            isbn = str(request.data.get('ISBN', ''))
+            if isbn:
+                booklist = Book(ISBN=isbn)
+                booklist.save()
+                response = jsonify({
+                    'ISBN': booklist.ISBN,
+                    'title': booklist.title,
+                    'author': booklist.author,
+                    'edition': booklist.edition,
+                    'publisher': booklist.publisher,
+                    'copies': booklist.copies
+                })
+                response.status_code = 201
+                return response
+
+        else:
+            booklists = Booklist.get_all()
+            results = []
+
+            for booklist in booklists:
+                obj = {
+                    'isbn': booklist.isbn,
+                    'title': booklist.title,
+                    'author': booklist.author,
+                    'edition': booklist.edition,
+                    'publisher': booklist.publisher,
+                    'copies': booklist.copies
+                }
+                results.append(obj)
+            response = jsonify(results)
+            response.status_code = 200
+            return response
+
```

Comment:
Functions are separated by single empty lines and classes by two empty lines.

Reply:
Fixing this. Thank you